File: app/src/main/java/com/numero/sojodia/view/NotSwipeViewPager.java
package com.numero.sojodia.view;

import android.content.Context;
import android.support.v4.view.ViewPager;
import android.util.AttributeSet;
import android.view.MotionEvent;

public class NotSwipeViewPager extends ViewPager {

    private static boolean isSwipeEnable = false;

    public NotSwipeViewPager(Context context) {
        super(context);
    }

    public NotSwipeViewPager(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    @Override
    public boolean onTouchEvent(MotionEvent event) {
        return isSwipeEnable;
    }

    @Override
    public boolean onInterceptTouchEvent(MotionEvent event) {
        return isSwipeEnable;
    }
}

File: app/src/main/kotlin/com/numero/sojodia/view/NotSwipeViewPager.kt
package com.numero.sojodia.view

import android.annotation.SuppressLint
import android.content.Context
import android.util.AttributeSet
import android.view.MotionEvent
import androidx.viewpager.widget.ViewPager

class NotSwipeViewPager @JvmOverloads constructor(context: Context, attrs: AttributeSet? = null) : ViewPager(context, attrs) {

    @SuppressLint("ClickableViewAccessibility")
    override fun onTouchEvent(event: MotionEvent): Boolean {
        return SWIPE_ENABLE
    }

    override fun onInterceptTouchEvent(event: MotionEvent): Boolean {
        return SWIPE_ENABLE
    }

    companion object {
        private const val SWIPE_ENABLE = false
    }
}
